clixon/element.py

```diff
@@ -493,10 +518,8 @@ def __getattr__(self, key: str) -> Optional[dict]:
         matching_children = [x for x in self._children if x._name == key]
         if matching_children:
             if len(matching_children) == 1:
-                self.__dict__[key] = matching_children[0]
```

Comment:
This is already removed in another commit, can we exclude it from this one?

Reply:
Removed from the commit.